src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -548,7 +553,7 @@ protected virtual void NoteWrite(Symbol variable, BoundExpression value, bool re
                 _writtenVariables.Add(variable);
                 if ((object)_sourceAssembly != null && variable.Kind == SymbolKind.Field)
                 {
-                    var field = (FieldSymbol)variable.OriginalDefinition;
```

Comment:
>OriginalDefinition [](start = 54, length = 18)

What effect is this change going to have on WriteConsideredUse call below and is this the effect we want?